code/modules/gear_presets/corpses.dm

```diff
@@ -439,50 +379,35 @@
 		ACCESS_CIVILIAN_BRIG,
 		ACCESS_CIVILIAN_MEDBAY,
 		ACCESS_CIVILIAN_COMMAND,
+		ACCESS_WY_GENERAL,
+		ACCESS_WY_COLONIAL,
 		ACCESS_WY_MEDICAL,
-		ACCESS_WY_ENGINEERING,
 		ACCESS_WY_SECURITY,
+		ACCESS_WY_ENGINEERING,
+		ACCESS_WY_FLIGHT,
+		ACCESS_WY_RESEARCH,
+		ACCESS_WY_EXEC,
+		ACCESS_WY_PMC,
+		ACCESS_WY_PMC_TL,
+		ACCESS_WY_ARMORY,
+		ACCESS_WY_SECRETS,
 		ACCESS_WY_LEADERSHIP,
-		ACCESS_WY_COLONIAL,
-		ACCESS_WY_GENERAL,
-	)
-
-/datum/equipment_preset/corpse/bridgeofficer/johnson/load_gear(mob/living/carbon/human/new_human)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/waiter(new_human), WEAR_BODY)
-	. = ..()
-
-/datum/equipment_preset/corpse/bridgeofficer/johnson/load_name(mob/living/carbon/human/new_human, randomise)
-	new_human.change_real_name(new_human, "Johnson Telovin")
-
-//*****************************************************************************************************/
-
-/datum/equipment_preset/corpse/commander
-	name = "Corpse - Commanding Officer"
-	idtype = /obj/item/card/id/general
-	assignment = "Commanding Officer"
-	access = list(
-		ACCESS_CIVILIAN_PUBLIC,
-		ACCESS_CIVILIAN_LOGISTICS,
-		ACCESS_CIVILIAN_ENGINEERING,
-		ACCESS_CIVILIAN_RESEARCH,
-		ACCESS_CIVILIAN_BRIG,
-		ACCESS_CIVILIAN_MEDBAY,
-		ACCESS_CIVILIAN_COMMAND,
+		ACCESS_WY_SENIOR_LEAD,
 	)
 
-/datum/equipment_preset/corpse/commander/load_gear(mob/living/carbon/human/new_human)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/rank/centcom_commander(new_human), WEAR_BODY)
+/datum/equipment_preset/corpse/administrator/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/liaison_suit/ivy(new_human), WEAR_BODY)
 	if(SSmapping.configs[GROUND_MAP].environment_traits[MAP_COLD])
```

Comment:
No longer necessary to have this if check; you can just always `add_ice_colony_survivor_equipment(new_human)` because it has its own test for the MAP_COLD trait

Reply:
Done